.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[octoprint.messages]
+[o:octoprint:p:octoprint:r:messages]
```

Comment:
Could you please explain what this is supposed to achieve?

Reply:
The Transifex client has been updated. Using an outdated configuration file will result in an error.
<https://developers.transifex.com/docs/migrating-from-older-versions-of-the-client>

Reply:
Ah, gotcha!